src/management-system-v2/prisma/schema.prisma

```diff
@@ -0,0 +1,126 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id               String      @id @default(uuid())
+  firstName        String
+  lastName         String
+  username         String
+  email            String      @unique
+  guest            Boolean
+  emailVerified    Boolean
+  processes        Process[]
+  ownedWorkspaces  Workspace[] @relation("WorkspaceOwner")
+  memberWorkspaces Workspace[] @relation("WorkspaceMember")
+  folders          Folder[]
+  Role             Role[]
+}
+
+model Process {
+  id                   String       @id @default(uuid())
+  originalId           String
+  name                 String
+  description          String
+  createdOn            DateTime
+  lastEdited           DateTime
+  type                 String
+  folder               Folder?      @relation(fields: [folderId], references: [id])
+  folderId             String?
```

Comment:
I think the folders needs to be optional because, while moving a process from one folder to another folder, there will be a moment where a process is not assigned to a folder,

Reply:
But that would not be the case in the DB. If at all, it would be only optional on the client, which this schema does not describe.

Reply:
ya, now I realised. The way I was implementing the move Process to Folder was first disconnecting the process from the oldFolder and then again connecting to the newFolder. For this to work, the folderId has to be nullable. 

I could also directly update the oldFolderId with the newFolderId. For this the folderId does not have to be nullable